api/onnx_web/chain/tile.py

```diff
@@ -7,7 +7,14 @@
 import numpy as np
 from PIL import Image
 
-from ..image.noise_source import noise_source_histogram
+from ..image.noise_source import (
+    noise_source_fill_edge,
```

Comment:
Linter is yelling at me for these and for the life of me I cannot remember if these need to be here or not given line 243

Reply:
https://github.com/ssube/onnx-web/blob/main/api/onnx_web/server/api.py#L261 should have fetched the correct function, so you only need to import the default

Reply:
Fixed